packages/tests/src/run-tests.js

```diff
@@ -0,0 +1,54 @@
+import { exec, spawn } from "child_process";
+import { promisify } from "util";
+
+import debug from "debug";
+
+const log = debug("waku:test:run");
```

Comment:
Now it should be safe to use `console.log` since this is merged https://github.com/waku-org/js-waku/pull/1291